src/style.css

```diff
@@ -0,0 +1,126 @@
+
+.block {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__img {
+  margin-top: 32px;
+  display: flex;
+}
+
+.img {
+  height: 134px;
+  width: 160px;
+}
+
+.card__information {
+  margin: 40px 16px 0;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+  align-self: center;
+  padding-bottom: 4px;
+}
+
+.card__code-num {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: rgba(97, 96, 112, 1);
+  padding-bottom: 16px;
+}
+
+.stars-reviews {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars-reviews__reviews {
+  color: rgba(6, 11, 53, 1);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.price__text {
+  color: rgba(97, 96, 112, 1);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+}
+
+.price__num {
+  color: rgba(6, 11, 53, 1);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__link {
+  align-self: center;
+  height: 40px;
+  width: 166px;
+  border-radius: 5px;
+  background: rgba(0, 172, 220, 1);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 16px auto;
+  text-decoration: none;
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.card__link:hover {
+  background-color: white;
+  border: 1px solid rgba(0, 172, 220, 1);
```

Comment:
Please add for prevent increasing the general width when adding a border
* {
  box-sizing: border-box
}

Reply:
Not fixed. Also add same recomendation upper